Triphouse-project/style.css

```diff
@@ -0,0 +1,224 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+/* roboto-regular - latin */
+@font-face {
+  font-family: "Roboto";
+  font-style: normal;
+  font-weight: 400;
+  src: url("./fonts/roboto-v27-latin-regular.eot"); /* IE9 Compat Modes */
+  src: local(""),
+    url("./fonts/roboto-v27-latin-regular.eot?#iefix")
+      format("embedded-opentype"),
+    /* IE6-IE8 */ url("./fonts/roboto-v27-latin-regular.woff2") format("woff2"),
+    /* Super Modern Browsers */ url("./fonts/roboto-v27-latin-regular.woff")
+      format("woff"),
+    /* Modern Browsers */ url("./fonts/roboto-v27-latin-regular.ttf")
+      format("truetype"),
+    /* Safari, Android, iOS */
+      url("./fonts/roboto-v27-latin-regular.svg#Roboto") format("svg"); /* Legacy iOS */
+}
+
+body {
+  width: 1440px;
```

Comment:
The body is whole page, add to html wrapper for header, main, footer and add to him with.
.wrapper {width: '1440px'}

Reply:
Added wrapper for header, main and footer.